bioconda_utils/pkg_test.py

```diff
@@ -79,7 +84,8 @@ def test_package(
     name_override=None,
     channels=["conda-forge", "defaults"],
     mulled_args="",
-    base_image=None
+    base_image=None,
+    conda_image=MULLED_CONDA_IMAGE,
```

Comment:
cosmetic: leading comma

Reply:
Ohhh, I like my trailing commas :wink:. See, if it would have been there in the first place, the diff would only be one-line and only touch what was added (or deleted in another case). I think we have some mixed styles in here; some time in the future we can unify that and with some linter assistance :wink: